src/System.Net.HttpListener/src/System/Net/HttpListener.cs

```diff
@@ -112,6 +112,158 @@ public HttpListenerPrefixCollection Prefixes
             }
         }
 
+        internal void AddPrefix(string uriPrefix)
+        {
+            if (NetEventSource.IsEnabled) NetEventSource.Enter(this, $"uriPrefix:{uriPrefix}");
+            string registeredPrefix = null;
+            try
+            {
+                if (uriPrefix == null)
+                {
+                    throw new ArgumentNullException(nameof(uriPrefix));
+                }
+                CheckDisposed();
+                if (NetEventSource.IsEnabled) NetEventSource.Info(this, "uriPrefix:" + uriPrefix);
```

Comment:
Nit: could use string interpolation as was done above, though I actually think this whole line could be removed, as it's not adding any additional information to what was already traced in the initial Enter call